### PR TITLE
Small issue fixes of the position button in RowName/Numbers dialog

### LINKED ISSUE #6254: Minor Issue with the position button in Row Numbers/Names dialog
The position button is not aligned in Prepare>Data Frame> Row Numbers/Names. Click on the first option (Copy Row Names Into Columns).

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #6254.
@africanmathsinitiative/developers and @rdstern this ready to test. Thanks.

### CODE REVIEW

File: instat/ucrSave.vb [hunk 1]
```diff
@@ -495,7 +495,7 @@ Public Class ucrSave
         ucrInputComboSave.Enabled = ucrChkSave.Checked
         ucrInputTextSave.Visible = False
         ucrInputComboSave.Visible = False
-        btnColumnPosition.Visible = ucrChkSave.Checked
+        btnColumnPosition.Visible = False
```
Comment (Patowhiz) [inline instat/ucrSave.vb:498]: @N-thony this looks good, however it not only fixes the bug but also reverts the key change made in #6230 that address the issue. Have a look at PR #6255 on how I have fixed the bug. Thanks.

Comment (N-thony) [reply instat/ucrSave.vb:498]: > @N-thony this looks good, however it not only fixes the bug but also reverts the key change made in #6230 that address the issue. Have a look at PR #6255 on how I have fixed the bug. Thanks.

@Patowhiz thanks. This bug was already fixed in PR #6255. So no need to fix it again here.

**Comment by lloyddewit** [comment]
@N-thony I can't see any changed files to review, please could you check? thanks

**Comment by N-thony** [comment]
@lloyddewit sorry, I think it is okay now.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern Please could you test with a selection of dialogs? thanks

**Comment by rdstern** [comment]
This seems to work for those dialogues.

There was another problem with this control that it was (mistakenly) implemented for all saving, including models, etc - see below.  That was fixed by @Patowhiz last week.  It is a problem here and I hope won't undo Patrick's fix when merged.

image

**Review by rdstern — APPROVED** [review]
I am still approving in the hope my comment below won't occur.  I'll check again after merging.

**Comment by N-thony** [comment]
> This seems to work for those dialogues.
> 
> There was another problem with this control that it was (mistakenly) implemented for all saving, including models, etc - see below. That was fixed by @Patowhiz last week. It is a problem here and I hope won't undo Patrick's fix when merged.
> 
> image

Yes @rdstern this will not undo Patrick's fix.

**Comment by dannyparsons** [comment]
There should only be 1 label for a PR. I made it "bug" since it seems to be a functional change to a dialog. Feel free to change if this isn't correct.

**Comment by lloyddewit** [comment]
@dannyparsons Understood. 
For this PR, 'skip-releasenotes' may be better because this PR corrects a bug that was inadvertently introduced by another PR in the same release. This PR just cancels that out so it's unlikely to be of interest to someone reading the release notes.

**Comment by dannyparsons** [comment]
ok thanks, I'll swap then.

**Pull request merged**